### DISCUSSION TITLE
How to setup DeeZy ?

### DISCUSSION THREAD

**Discussion by AdithyanNellickal** [staxrip/staxrip#1107] [Q&A] [answered]
Hi,
I was using Staxrip 2.13.0 and just downloaded the latest version today. Saw the option for DeeZy and tried to check, and it showed that it wasn't installed. But on trying to download, it's redirecting to Dolby audio's site. And I couldn't see a download option. 

Can someone please help me in setting up DeeZy ? Thanks in advance.

**Comment by Dendraspis** [comment, marked as answer]
Hello,
`DeeZy` is distributed with StaxRip and doesn't require any adjustments, installation or so. It's a wrapper for `DEE`, which is a proprietary non-free app from Dolby, made for professional usage. So in order to use `DeeZy`, you have to own `DEE` and set the path to `DEE.exe`.

For further information see also:
https://github.com/jlw4049/DeeZy 
https://professional.dolby.com/product/media-processing-and-delivery/dee--dolby-encoding-engine/

**Reply by AdithyanNellickal** [reply]
Bro, I do use DEE cracked one as separate. Can I use it with Stax ? 
![IMG_20230901_102939](https://github.com/staxrip/staxrip/assets/60499868/f0974e14-fcb9-4d32-9ed3-82a0fca7733f)

**Reply by Dendraspis** [reply]
`DEE` yes,  `DEEW` indirectly, but you can use `DeeZy` intead.

Just go to `Apps » Manage`, search for `DEE`, then click on `Tools » Edit Path` and you're ready to use `DeeZy`.
